libsolidity/codegen/LValue.cpp

```diff
@@ -206,6 +206,12 @@ void StorageItem::retrieveValue(SourceLocation const&, bool _remove) const
 				CompilerUtils(m_context).splitExternalFunctionType(false);
 				cleaned = true;
 			}
+			else if (fun->kind() == FunctionType::Kind::Internal)
+			{
+				m_context << Instruction::DUP1 << Instruction::ISZERO;
+				CompilerUtils(m_context).pushZeroValue(*fun);
```

Comment:
What is the zero value in this case? A function that reverts?

Reply:
A function that has an invalid opcode. After what I have learnt now, it might actually be more efficient to just use the literal zero - it will result in a bad jumpdest and not in an invalid opcode, though.

Reply:
Could this be used also for the other recent invalid function bug then?

Reply:
The other recent bug was inside `pushZeroValue` as far as I remember. So if that is the case, then it was not a bug after all.